marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/test-data/mappings/PersonMapping/PersonMapping-7.mapping.json

```diff
@@ -0,0 +1,24 @@
+{
+  "lang": "zxx",
+  "name": "PersonMapping",
+  "version": 7,
+  "sourceContext": "/",
+  "targetEntityType": "http://marklogic.com/data-hub/example/Person-1.0.0/Person",
+  "properties": {
+    "id": {
+      "sourcedFrom": "personId"
+    },
+    "nickname": {
+      "sourcedFrom": "$URI"
```

Comment:
@rjrudin ,
Can "sourcedFrom" be set to something like concat($URI, 'test-uri') ? If so, adding a test for that would be good.

Reply:
I don't think it would prove anything additional. The contents of sourcedFrom are evaluated by ML, and thus from a DHF testing perspective, there's no difference between "$URI" and an expression that incorporates multiple functions and predicates. Either "$URI" gets resolved, or it doesn't - the complexity of the expression doesn't affect that.